client/my-sites/theme/theme-style-variations/style.scss

```diff
@@ -14,12 +14,12 @@
 			height: 100%;
 		}
 
-		.theme__sheet-style-variations-previews {
+		.theme__sheet-style-variations-previews .global-styles-variations {
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

What is this `.global-styles-variations`?
Do we always need this class to override `@automattic/global-styles`?

Reply:
`.global-styles-variations` is from `@automattic/global-styles` and it's the wrapper class to all the style previews. In this case, I think  `.theme__sheet-style-variations-previews` is redundant, and we can use `.global-styles-variations` is enough 🙂

Reply:
>.global-styles-variations is from @automattic/global-styles and it's the wrapper class to all the style previews.

Thank you, I understand!